homeassistant/components/google_assistant/smart_home.py

```diff
@@ -168,6 +169,11 @@ def query_serialize(self):
         for trt in self.traits():
             if trt.can_execute(command, params):
                 await trt.execute(command, params)
+                self.hass.bus.fire(EVENT_COMMAND_RECEIVED, {
```

Comment:
```suggestion
                self.hass.bus.async_fire(EVENT_COMMAND_RECEIVED, {
```

Reply:
I don't know if this is the right location. We won't see any SYNC commands come in and we will fire 3 events if a command intent has 3 commands in it. It might be better to move it up into the main handler.